client/src/components/PropertiesList.tsx

```diff
@@ -34,39 +53,68 @@ const PropertiesListWithoutI18n: React.FC<
 
   const addrs = props.state.context.portfolioData.assocAddrs;
   const rsunitslatestyear = props.state.context.portfolioData.searchAddr.rsunitslatestyear;
+
   return (
     <div className="PropertiesList">
-      <ReactTable
+      <ReactTableFixedColumns
         data={addrs}
         minRows={Browser.isMobile() ? 5 : 10}
         defaultPageSize={addrs.length}
         showPagination={false}
         columns={[
           {
-            Header: i18n._(t`Location`),
+            fixed: "left",
             columns: [
               {
-                Header: i18n._(t`Address`),
+                Header: (
+                  <>
+                    <Trans>Address</Trans>
+                    <ArrowIcon />
+                  </>
+                ),
                 accessor: (d) => `${d.housenumber} ${d.streetname}`,
                 id: "address",
-                minWidth: 150,
+                minWidth: 130,
+                fixed: "left",
                 style: {
                   textAlign: "left",
+                  whiteSpace: "unset",
+                  paddingRight: "0.75rem",
                 },
               },
+            ],
+          },
+          {
+            Header: i18n._(t`Location`),
+            columns: [
               {
-                Header: i18n._(t`Zipcode`),
+                Header: (
+                  <>
+                    <Trans>Zipcode</Trans>
+                    <ArrowIcon />
+                  </>
+                ),
                 accessor: (d) => d.zip,
                 id: "zip",
-                width: 75,
+                width: i18n.language === "es" ? 125 : 85,
```

Comment:
I am curious if these kinds of conditionals would be better expressed as e.g. some kind of type, e.g.:

```typescript
type LocaliedLabelWidths = {
  zipcode: number,
  borough: number,
  /* Et cetera... */
};
```

then the localized metrics could just be passed into this function and the `width` could just be referred to by `widths.zipcodeWidth` etc, which would make it really easy to add more locales in the future... Hmm.  I also wonder if the width could just be a function of the actual number of characters in the localized string... anyhoo, just some thoughts.

Reply:
@toolness I tried doing your latter recommendation of making width a function of label length! check it out